RELEASE_NOTES.md

```diff
@@ -10,6 +10,9 @@ v1.0.0-beta.1.x
   `TraitsData`.
   [#1153](https://github.com/OpenAssetIO/OpenAssetIO/issues/1153)
 
+- Modified no-argument constructor of `Context` to create an empty (but
```

Comment:
[consider] I'd also perhaps argue this is an improvement or probably even a bugfix too as it was never meant to null before.

Reply:
I like bug fix, that seems more appropriate to both changes.  Updated in 7789fe6b